swagger.yaml

```diff
@@ -721,15 +721,15 @@ paths:
                 items:
                   $ref: "#/components/schemas/Error"
 
-  "/severa/{severaGuid}/flextime":
+  "/severa/{severaUserGuid}/flextime":
```

Comment:
This needs to be changes to other places also. Where it now severaGuid. In example in keycloak-Api-service. Maybe it on bugfix PR ?

Reply:
I thought we discussed the idea of renaming all references to a Guid to Id. I think this is the best options. So in this case it should be severaUserId. Please ensure all other references to Guid are upated

Reply:
I won't leave any other comments relating to this change so please check